modules/pref/src/main/RequestPref.scala

```diff
@@ -32,7 +32,8 @@ object RequestPref:
         is3d = paramOrSession("is3d").has("true"),
         board = default.board.copy(
           opacity = paramOrSession("boardOpacity").flatMap(_.toIntOption) | default.board.opacity,
-          brightness = paramOrSession("boardBrightness").flatMap(_.toIntOption) | default.board.brightness
+          brightness = paramOrSession("boardBrightness").flatMap(_.toIntOption) | default.board.brightness,
+          hue = paramOrSession("boardHue").flatMap(_.toIntOption) | default.board.hue
```

Comment:
I had removed that on purpose. Because I'm not sure hue is a useful thing to support, I'd rather not add it to the public API right away.

Reply:
Public API? Isn't it just so anons can have persistent values?